src/main/java/com/google/flourbot/api/Bot.java

```diff
@@ -77,15 +77,13 @@ public void onEvent(@RequestBody JsonNode event) throws IOException, GeneralSecu
         String email = event.at("/message/sender/email").asText();
         String message = event.at("/message/text").asText();
         // Sends request to execution module
-        replyText = exec.execute(email, message);
+        replyText = macroExecutionModule.execute(email, message);
         break;
       case "REMOVED_FROM_SPACE":
-        String name = event.at("/space/name").asText();
-        logger.info(String.format("Bot removed from %s", name));
+        logger.info("Bot removed from space");
         break;
       default:
-        replyText = "Cannot determine event type";
-        break;
+        throw new IllegalArgumentException();
```

Comment:
Add "Unrecognized event type: "+event.at("/type").asText() as the argument to the exception.

Reply:
resolved